_rules/iframe-non-empty-accessible-name-cae760.md

```diff
@@ -17,13 +17,17 @@ acknowledgments:
   authors:
     - Jey Nandakumar
     - Wilco Fiers
+    - Tom Brunet
   funding:
     - WAI-Tools
 ---
 
 ## Applicability
 
-This rule applies to `iframe` elements that are [included in the accessibility tree][] and do not have a negative `tabindex` [attribute value][].
+This rule applies to `iframe` elements that are [included in the accessibility tree][] and for which all of the following are true:
+- does not have a negative `tabindex` [attribute value][]; and
+- does not have a `role` [attribute value][] of `presentation`; and
+- does not have a `role` [attribute value][] of `none`.
```

Comment:
I would suggest this:

```suggestion
- the `iframe` does not have a negative `tabindex` [attribute value][]; and
- the `iframe` does not have an [explicit semantic role][] of `presentation` or `none`.
```

Restated to follow the conditional list writing style, and then using explicit semantic role, so we don't require an exact attribute value match, to account for whitespace and fallback roles.